parser/parser.y

```diff
@@ -6920,7 +6919,7 @@ FunctionNameConflict:
 |	"DAY"
 |	"HOUR"
 |	"IF"
-|	"INTERVAL" %prec lowerThanIntervalKeyword
```

Comment:
Don't know why we set %prec before.
Follow the  [MySQL](https://github.com/mysql/mysql-server/blob/3290a66c89eb1625a7058e0ef732432b6952b435/sql/sql_yacc.yy#L1416) to fix it.

Reply:
FYI, https://github.com/pingcap/tidb/pull/2370, this is where it is introduced.